.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "libkineto/third_party/dynolog"]
 	path = libkineto/third_party/dynolog
 	url = https://github.com/facebookincubator/dynolog.git
+[submodule "libkineto/third_party/folly"]
```

Comment:
Since pytorch/kineto is a submodule of pytorch/pytorch, adding facebook/folly would also make folly a third-party submodule of pytorch/pytorch.

We may want to remove the dependency on facebook/folly for CuptiActivityProfilerTest, since I think that is the only test that uses folly afaik. Maybe we can skip it for now. cc @malfet , @albanD.

Reply:
We definitely do not want to take folly as a dependency in PyTorch.

Reply:
Ok,  I will try to remove facebook/folly.